src/eval/methods.rs

```diff
@@ -154,6 +154,7 @@ pub fn call(
                 match method {
                     "page" => vm.vt.introspector.page(location).into(),
                     "position" => vm.vt.introspector.position(location).into(),
+                    "numbering" => vm.vt.introspector.page_numbering(location),
```

Comment:
```suggestion
                    "page-numbering" => vm.vt.introspector.page_numbering(location),
```

Reply:
can you also add an entry in `docs/src/reference/types.md`? this part of the docs system is garbage, I know.

Reply:
Let's just say that we'll need to work on that :D